pkg/version/version.go

```diff
@@ -47,3 +48,7 @@ func GetOpenShiftVersion() string {
 func GetOpenShiftSemverVersion() (semver.Version, error) {
 	return semver.Make(strings.TrimPrefix(GetOpenShiftVersion(), VersionPrefix))
```

Comment:
@LalatenduMohanty here is your code which makes use of semantic versioning. You added your code right below ;-) This is using https://github.com/blang/semver which allows you to compare versions and determine whether they are compatible in a semantic versioning sense.